### DISCUSSION TITLE
Is types.h DccControl.bcd unused?

### DISCUSSION THREAD

**Discussion by KC5VDJ** [kej715/DtCyber#151] [General]
Subject line says it all.

I recall reading that this might be a limited case with specific equipment (bcd<->internal).

**Comment by kej715** [comment]
If you are referring to the bcd field of the DccControl structure defined
in dcc6681.c (not types.h), then this field is set when certain function
codes are issued to the DCC, but it seems that the field is not currently
read anywhere in DtCyber. The same is true of the ios field.
